cypress/e2e/tests/home.cy.js

```diff
@@ -0,0 +1,509 @@
+context('Numbers Product SVG Link on Homepage', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('svg[alt="Numbers"]')
+      .click()
+  })
+
+  it('cy.go() - go back or forward in the browser\'s history', () => {
+
+    cy.location('pathname').should('include', '/docs/numbers')
+
+    cy.go('back')
+    cy.location('pathname').should('not.include', '/docs/numbers')
+
+    cy.go('forward')
+    cy.location('pathname').should('include', '/docs/numbers')
+
+    // clicking back
+    cy.go(-1)
+    cy.location('pathname').should('not.include', '/docs/numbers')
+
+    // clicking forward
+    cy.go(1)
+    cy.location('pathname').should('include', '/docs/numbers')
+  })
+
+  it('cy.reload() - reload the page', () => {
+    cy.reload()
+
+    // reload the page without using the cache
+    cy.reload(true)
+  })
+
+})
+
+context('Numbers Product Text Link on Homepage', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('a')
+      .contains('Numbers')
+      .click()
+  })
+
+  it('cy.go() - go back or forward in the browser\'s history', () => {
+
+    cy.location('pathname').should('include', '/docs/numbers')
+
+    cy.go('back')
+    cy.location('pathname').should('not.include', '/docs/numbers')
+
+    cy.go('forward')
+    cy.location('pathname').should('include', '/docs/numbers')
+
+    // clicking back
+    cy.go(-1)
+    cy.location('pathname').should('not.include', '/docs/numbers')
+
+    // clicking forward
+    cy.go(1)
+    cy.location('pathname').should('include', '/docs/numbers')
+  })
+
+  it('cy.reload() - reload the page', () => {
+    cy.reload()
+
+    // reload the page without using the cache
+    cy.reload(true)
+  })
+
+  it('visits /docs/numbers directly', () => {
+
+    cy.visit('/docs/numbers', {
+      timeout: 50000, // increase total time for the visit to resolve
+      onBeforeLoad (contentWindow) {
+        // contentWindow is the remote page's window object
+        expect(typeof contentWindow === 'object').to.be.true
```

Comment:
Please explain, not sure what checking type of `contentwindow` means!

Reply:
This is checking that the window object exists. So that the page exists both before all the components load and after. It was an example of what should be checked.